.github/actions/synthetics-stack-e2e-run/action.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: synthetics-stack-e2e-run
+description: |
+  This action is part of the synthetics-stack-e2e action.
+
+  It first creates a Kibana docker image, then it deploys an ESS cluster with the created docker image,
+  and finally it runs the Synthetics Stack E2E tests against the deployed ESS cluster.
+
+inputs:
+  buildkite-pipeline-slug:
+    description: 'The Buildkite pipeline slug'
+    default: kibana-pr-synthetics-stack-e2e-ci
+    required: false
```

Comment:
I'm not sure whether this is required. Consumers don't need to know whats the underlying Buildkite pipeline IMO

Reply:
It was easy to change the pipeline for testing.. but true.. I don't see that this will change anymore.. Now I can just use a different branch

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2367/commits/536df233aa413b6ff81564a94e27340c5d27b395